src/ui/components/CCDiceMenu.vue

```diff
@@ -298,10 +299,17 @@ export default Vue.extend({
   }),
   mounted() {
     this.reset()
+    if (this.autoroll) this.$nextTick(this.autoRoll)
   },
   watch: {
     menu() {
-      this.reset()
+      if (!this.menu || !this.autoroll) this.reset()
```

Comment:
This change will allow someone to click on the dice icon in order to see how that roll was achieved. This change didn't feel good to me until I made this because I was just getting top level numbers and couldn't see what went in to it if I wanted to.

Reply:
https://user-images.githubusercontent.com/6234658/128186423-ac813ab6-91ec-456c-8a23-a85bb204e0f5.mp4